Features/links.feature

```diff
@@ -0,0 +1,572 @@
+    Dan bevat de response het voorkomen van het object
+    En wordt de self link van de resource (het voorkomen) opgebouwd zoals beschreven in scenario 'self link van een resource'
+    En worden de links naar gerelateerde resources opgebouwd zoals beschreven in scenario 'link naar gerelateerde resources in een resource voorkomen (niet zijnde een woonplaats'
+    En worden er geen paginering links opgegeven zoals beschreven in scenario 'geen paginering links bij levenscyclus'
+
+    {
+      "nummeraanduiding": {
+        ...
+      },
+      "_links": {
+        "self": {
+          "href": "https://api.bag.kadaster.nl/lvbag/individuelebevragingen/v2/nummeraanduidingen/<nummeraanduidingIdentificatie/<versie>/<timestamp tijdstip registratie LV>"
+        },
+        "ligtInWoonplaats": {
+          "href": "https://api.bag.kadaster.nl/lvbag/individuelebevragingen/v2/woonplaatsen/<woonplaatsIdentificatie>/lvc"
+        },
+        "ligtAanOpenbareRuimte": {
+          "href": "https://api.bag.kadaster.nl/lvbag/individuelebevragingen/v2/openbareruimten/<openbareRuimteIdentificatie>/lvc"
+        }
+      }
+    }
+
+  Scenario: links naar resources bij voorkomen endpoints (zijnde een woonplaats)
+    Als er een object (zijnde een woonplaats) voorkomen wordt opgevraagd op het voorkomen endpoint van het betreffende object type
+    Dan bevat de response het voorkomen van het object
+    En wordt de self link van de resource (het voorkomen) opgebouwd zoals beschreven in scenario 'self link van een resource'
+    En worden de links naar gerelateerde resources opgebouwd zoals beschreven in scenario 'link naar gerelateerde resources in een resource voorkomen (zijnde een woonplaats'
+    En worden er geen paginering links opgegeven zoals beschreven in scenario 'geen paginering links bij levenscyclus'
+
+    {
+      "woonplaats": {
+        ...
+      },
+      "_links": {
+        "self": {
+          "href": "https://api.bag.kadaster.nl/lvbag/individuelebevragingen/v2/woonplaatsen/<woonplaatseIdentificatie/<versie>/<timestamp tijdstip registratie LV>"
+        },
+        "bronhouders": [
+          {
+            "href": "https://api.bag.kadaster.nl/lvbag/individuelebevragingen/v2/bronhouders/<bronhouderIdentificatie>/<versie>/<timestamp tijdstip registratie LV>"
+          },
+        ]
+      }
+    }
+
+  Scenario: geen expand van gerelateerde resource van een resource bij voorkomen endpoints
+    Als er een object voorkomen wordt opgevraagd op het voorkomen endpoint van het betreffende object type
+    En één, meer of alle gerelateerde resources worden geëxpand met de expand parameter
+    Dan bevat de response het voorkomen van het object
+    En wordt de self link van de resource (het voorkomen) opgebouwd zoals beschreven in scenario 'self link van een resource'
+    En worden de links naar gerelateerde resources opgebouwd zoals beschreven in scenario 'link naar gerelateerde resources in een resource'
```

Comment:
Hier moet denk ik nog komen dat in een voorkomen endpoint de links naar gerelateerde resources moeten verwijzen naar het levenscyclus endpoint van die resource? (maar misschien heb je dat al aangepast maar nog niet gecommit, dan is dit een reminder)

Reply:
Aangepast.